packages/material-renderers/package.json

```
@@ -87,7 +87,7 @@
     "@emotion/styled": "^11.3.0",
     "@jsonforms/core": "3.1.1-alpha.0",
     "@jsonforms/react": "3.1.1-alpha.0",
-    "@mui/icons-material": "~5.11.16",
+    "@mui/icons-material": "^5.11.16 || ^6.0.0",
```

I quickly looked it up now and saw that there is no version 6.x of [@mui/icons-material](https://www.npmjs.com/package/@mui/icons-material?activeTab=versions) at the moment. Thus, I would not include it because we don't know if it will break.

```suggestion
    "@mui/icons-material": "^5.11.16",
```